src/path_info.cpp

```diff
@@ -111,9 +111,9 @@ void PATH_INFO::set_standard_filenames(void)
 {
     // Special: data_dir lua_dir and gfx_dir
     if (!FILENAMES["base_path"].empty()) {
-        update_pathname("datadir", FILENAMES["base_path"] + "share/cataclysm-dda/");
```

Comment:
This is trying to comply with a linux installer type layout, assuming base_path is /usr/ or similar, that functionality needs to remain, though I don't think it's a problem if it requires an extra option or something.